pkg/apply/applier_apply_options.go

```diff
@@ -0,0 +1,141 @@
+// Copyright 2021 The Kubernetes Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package apply
+
+import (
+	"time"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/cli-utils/pkg/apply/event"
+	"sigs.k8s.io/cli-utils/pkg/common"
+	"sigs.k8s.io/cli-utils/pkg/inventory"
+)
+
+const defaultPollInterval = 2 * time.Second
+
+type ApplierOptions struct {
```

Comment:
Do we need to change the name to ApplierOptions? It seems a bit verbose given that it is in the apply package.

Reply:
I didn't see the discussion around this until now. I'm ok with the change.